src/amp/components/topMeta/Branding.tsx

```diff
@@ -61,3 +61,56 @@ export const Branding: React.FC<{
 		</div>
 	);
 };
+
+export const Branding: React.FC<{
+	commercialProperties: CommercialProperties;
+	pillar: Theme;
+}> = ({ commercialProperties, pillar }) => {
+	const brandingStyles = css`
+		display: none;
+	`;
+	const ukStyles = css`
+		.amp-iso-country-gb & {
+			display: block;
+		}
+	`;
+	const usStyles = css`
+		.amp-iso-country-us & {
+			display: block;
+		}
+	`;
+	const auStyles = css`
+		.amp-iso-country-au & {
+			display: block;
+		}
+	`;
+	const intStyles = css`
+		.amp-geo-group-eea:not(.amp-iso-country-gb) &,
+		.amp-geo-no-group & {
+			display: block;
+		}
+	`;
+	const editionStyles: Record<Edition, SerializedStyles> = {
+		UK: ukStyles,
+		US: usStyles,
+		AU: auStyles,
+		INT: intStyles,
+	};
```

Comment:
This is pretty generic stuff, and it would be good to see them extracted to a single file, or even a wrapper component that generates all the variations automatically.

Most of these classes (except for `RoW`) are identical to: https://github.com/guardian/dotcom-rendering/blob/bc8c9c75aa3556fdf0523576f37d5035eaaa2c26/src/amp/components/Ad.tsx#L106-L135

Reply:
Really good point, if its not a blocker I'll have a think about the best way to extract these classes (or create a wrapper component) and raise a follow-up PR with those changes.